src/stores/FileBrowserStore.ts

```diff
@@ -347,6 +348,26 @@ export class FileBrowserStore {
                 this.startingDirectory = this.fileList.directory;
             }
         }
+        if (PreferenceStore.Instance.keepLastUsedFolder) {
```

Comment:
`const preferences = PreferenceStore.Instance` and reuse that within the function